### ISSUE TITLE
Suggestion of how to save 2D detector data 

### ISSUE THREAD

**Issue by RGBYCP** [alexbjorling/contrast#3]
Hi,
instead of having /measurement/detector_name, I suggest to unify these entries for all 2D detectors, i.e. Eiger and Pilatus.
Example /measurement/2Ddetector and place the information of the type of detector elsewhere in the HDF file format.
All of this is 2D detector data, why name it differently in the HDF5? Is it possible, that one experiment will have one HDF5 file containing data from two different 2D detectors?
Thank you.

**Comment by kahntm** [comment]
Yes. It is not uncommon to have two or more 2D detectors active in the same scan.
